tests/test.js

```diff
@@ -438,4 +438,174 @@ test('NbsDecoder.getPackets() returns the last packet of each type when given a
   });
 });
 
+const multiTypeNextTimestampArray = [
+  { type: pongType, subtype: 0 },
+  { type: pingType, subtype: 0 },
+  { type: pangType, subtype: 100 },
+  { type: pangType, subtype: 200 },
+];
+
+const [start, end] = decoder.getTimestampRange({ type: pingType, subtype: 0 });
+const timestampBeforeStart = { seconds: 0, nanos: 0 };
+const timestampAfterEnd = { seconds: 1900, nanos: 0 };
+
+test('NbsDecoder.nextTimestamp() outputs the timestamp of the most recent message when given steps = 0', () => {
+  const new_timestamp = decoder.nextTimestamp(
```

Comment:
Use camel case in JS/TS

```suggestion
  const newTimestamp = decoder.nextTimestamp(
```

Reply:
Changed.